### PR TITLE
8 add logos

### PR DESCRIPTION
**Pull request by MeerIsmailAli**
added logos for the sponsor page please review

### CODE REVIEW

**Comment by anirudhprabhakaran3** [comment]
Rebase with main once, a few changes have been made there.

File: web/templates/pages/committee.html [hunk 1]
```diff
@@ -906,9 +789,9 @@
     <p class="text-3xl text-gray-900 , text-left">Publicity Chairs</p>
 </div>
 
-<div id="PublicityChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="PublicityChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:793]: Do not use custom styles! Please use whatever is provided by Flowbite. If you want to add something custom, use Tailwind, as Flowbite is made on top of tailwind.

File: web/templates/pages/committee.html [hunk 2]
```diff
@@ -847,9 +730,9 @@
     <p class="text-3xl text-gray-900 , text-left">Conveners</p>
 </div>
 
-<div id="tutorialChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="tutorialChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:734]: Remove styles here also, use appropriate classes.

File: web/templates/pages/committee.html [hunk 3]
```diff
@@ -798,9 +681,9 @@
     <p class="text-3xl text-gray-900 , text-left">Plenary Chairs</p>
 </div>
 
-<div id="tutorialChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="tutorialChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:685]: Styles

File: web/templates/pages/committee.html [hunk 4]
```diff
@@ -700,9 +583,9 @@
     <p class="text-3xl text-gray-900 , text-left">Finance Chairs</p>
 </div>
 
-<div id="FinanceChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="FinanceChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:587]: Styles

File: web/templates/pages/committee.html [hunk 5]
```diff
@@ -749,9 +632,9 @@
     <p class="text-3xl text-gray-900 , text-left">Tutorial Chairs</p>
 </div>
 
-<div id="tutorialChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="tutorialChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:636]: Styles

File: web/templates/pages/committee.html [hunk 6]
```diff
@@ -652,9 +524,9 @@
     <p class="text-3xl text-gray-900 , text-left">Publication Chairs</p>
 </div>
 
-<div id="PublicationChairs">
-    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="display: flex;justify-content: center;align-items: center;width: 90vw;height: auto;margin: 0 auto;border: 0.5px solid #c2c0c0;">
-        <table class="w-full text-sm text-left text-gray-500 ">
+<div id="PublicationChairs" class="flex justify-center">
+    <div class="relative overflow-x-auto shadow-md sm:rounded-lg" style="border: 0.5px solid #c2c0c0;width: 90vw;">
```
Comment (anirudhprabhakaran3) [inline web/templates/pages/committee.html:528]: Styles

**Review by anirudhprabhakaran3 — CHANGES_REQUESTED** [review]

**Comment by MeerIsmailAli** [comment]
please review the changes in styles

**Comment by MeerIsmailAli** [comment]
in the sponsors section, I coudnt find a proper nitk .svg embelem i tried to convert one but it didnt come out clearly

**Comment by anirudhprabhakaran3** [comment]
For NITK, you can use this.
nitkwhite

**Comment by anirudhprabhakaran3** [comment]
Btw, try experimenting with SVG and PNG. If PNG works well enough, and is good on responsive mobiles, you can use that too.

**Review by anirudhprabhakaran3 — APPROVED** [review]

**Pull request merged**